code/__DEFINES/is_helpers.dm

```diff
@@ -249,6 +249,7 @@ GLOBAL_LIST_INIT(book_types, typecacheof(list(
 #define is_bartender_job(job_type) (istype(job_type, /datum/job/bartender))
 #define is_captain_job(job_type) (istype(job_type, /datum/job/captain))
 #define is_chaplain_job(job_type) (istype(job_type, /datum/job/chaplain))
+#define is_curator_job(job_type) (istype(job_type, /datum/job/curator)) //SKYRAT EDIT: Xenoarch and Glassblowing
```

Comment:
Don't we have a few files for Skyrat defines? I feel like this wouldn't really need to be here.

Reply:
resolved in upcoming commit